### DISCUSSION TITLE
epoch

### DISCUSSION THREAD

**Discussion by LiYan0306** [open-mmlab/mmdetection#8996] [Q&A]
怎么设置训练的epoch 啊？

**Comment by jbwang1997** [comment]
You can set the `may_epoch` at [here](https://github.com/open-mmlab/mmdetection/blob/master/configs/_base_/schedules/schedule_1x.py#L11)

**Comment by LiYan0306** [comment]
Thank you for your help!




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "open-mmlab/mmdetection"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年10月12日(星期三) 下午4:30
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [open-mmlab/mmdetection] epoch (Discussion #8996)





 
You can set the may_epoch at here
